templates/zilencer/emails/confirm_remoteserver.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html lang="en">
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+        <title>Zulip</title>
+    </head>
+    <body>
+        <table width="80%" style="align:center; max-width:800px" align="center">
+            <tr>
+                <td style="font-size:16px; font-family:Helvetica;">
+                    <p>Hi!</p>
```

Comment:
yeah, totally reasonable. In this case I think the hassle of maintaining two separate email pathways is probably not worth it.